lib/modules/datasource/docker/index.ts

```diff
@@ -630,16 +630,17 @@ export class DockerDatasource extends Datasource {
     let url: string | null =
```

Comment:
Maintainers, you should rename the PR title, if you decide to accept the change. Right now the PR title does not follow our rules.